src/components/AuthSecretKey.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from "react";
+import { Link, useHistory } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+import { Keypair } from "stellar-sdk";
+
+import { fetchAccount } from "ducks/account";
+
+const Warning = styled.div`
+  background-color: #f3e5e5;
+  color: #681e1e;
+  padding: 20px;
+  margin-bottom: 20px;
+`;
+
+const TempLink = styled(Link)`
+  display: block;
+  margin-bottom: 20px;
+`;
+
+const TempButton = styled.button`
+  margin-bottom: 20px;
+`;
+
+const TempInput = styled.input`
+  margin-bottom: 20px;
+  min-width: 300px;
+`;
+
+export const AuthSecretKey = () => {
+  const dispatch = useDispatch();
+  let history = useHistory();
+
+  const [canContinue, setCanContinue] = useState(false);
```

Comment:
By the same token, I think state variables should be named to express what they do. Here, this state variable controls whether or not they've agreed to the warning, but it's named as if it would be "true" if it's a valid entry. That's not technically true; `canContinue` can be set to true even if the input is empty. So I think you should be more specific here.